### ISSUE TITLE
Latest Update of Airtouch 5 breaks api? 

### ISSUE THREAD

**Issue by codeghari** [TheNoctambulist/hass-airtouch#4]
### System Health details

Logger: pyairtouch.comms.socket
Source: .local/lib/python3.12/site-packages/pyairtouch/at5/comms/xC023_ac_status.py:230
First occurred: 8:30:05 PM (11741 occurrences)
Last logged: 9:23:35 PM

Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 92 c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 0c 60
Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 94 c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 8c 76
Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 96 c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 33 85
Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 98 c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 cc 59
Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 9a c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 73 aa
Traceback (most recent call last):
  File "/config/.local/lib/python3.12/site-packages/pyairtouch/comms/socket.py", line 416, in _read_one_message
    message_result = message_decoder.decode(message_buffer, header)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/.local/lib/python3.12/site-packages/pyairtouch/at5/comms/xC0_ctrl_status.py", line 211, in decode
    sub_message_result = sub_message_decoder.decode(
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/.local/lib/python3.12/site-packages/pyairtouch/at5/comms/xC023_ac_status.py", line 230, in decode
    raise comms.DecodeError(
pyairtouch.comms.DecodeError: Header repeat_length (14) != AC Status Data size (10)

### Checklist

- [X] I have enabled debug logging for the integration.
- [X] This issue is not a duplicate issue of any [previous issues](https://github.com/thenoctambulist/hass-airtouch/issues?q=is%3Aissue+label%3A%22bug%22+)..

### Describe the issue

It appears that the latest update of Airtouch (released yesterday) has broken the integration.

Both the "official" integration and this custom one you developed seem to be unable to connect.

When either integration is attempting to connect, it looks like the console gets flooded with requests and stops working correctly until the attempts are stopped from home assistant.

Mobile app also stops working when the integrations are trying to connect (and failing).

Happy to help you with anything and a fair warning to anyone not to update to latest 😅

### Reproduction steps

1. Be on latest console update for airtouch 5
2. This should be service version 1.7 and console 1.2
3. integrations stop working and break Airtouch console and mobile applications while they are polling

### Debug logs

```Text
Logger: pyairtouch.comms.socket
Source: .local/lib/python3.12/site-packages/pyairtouch/at5/comms/xC023_ac_status.py:230
First occurred: 8:30:05 PM (11741 occurrences)
Last logged: 9:23:35 PM

Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 92 c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 0c 60
Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 94 c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 8c 76
Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 96 c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 33 85
Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 98 c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 cc 59
Error decoding bytes: 55 55 55 ab 00 00 00 22 00 22 55 55 55 aa b0 80 9a c0 00 16 23 00 00 00 00 0e 00 01 00 14 a0 c1 0a b7 00 00 d0 00 e2 bc 00 00 73 aa
Traceback (most recent call last):
  File "/config/.local/lib/python3.12/site-packages/pyairtouch/comms/socket.py", line 416, in _read_one_message
    message_result = message_decoder.decode(message_buffer, header)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/.local/lib/python3.12/site-packages/pyairtouch/at5/comms/xC0_ctrl_status.py", line 211, in decode
    sub_message_result = sub_message_decoder.decode(
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/.local/lib/python3.12/site-packages/pyairtouch/at5/comms/xC023_ac_status.py", line 230, in decode
    raise comms.DecodeError(
pyairtouch.comms.DecodeError: Header repeat_length (14) != AC Status Data size (10)
```

**Comment by pppaulie** [comment]
Broken for me too since the latest AirTouch update. 
I also noticed the AirTouch 5 Android phone APP has issues with fetching data and requires a close and re-open of the app on the console.  This only temporarily fixed the phone app as shortly afterwards, the error appeared again. 
![SmartSelect_20240627_063328_AirTouch 5](https://github.com/TheNoctambulist/hass-airtouch/assets/42607382/2d343a1c-1e9f-4e68-aa0d-3421ab46fe66)

**Comment by JohnMcIl** [comment]
Agreed!!! Broke for me also! 100% caused by the update.
![Screenshot 2024-06-27 at 10 35 45 am](https://github.com/TheNoctambulist/hass-airtouch/assets/3096550/57046bf7-49b2-4451-aff3-9a7d4ea75758)

Debug logging enabled…

@TheNoctambulist FYI

**Comment by pppaulie** [comment]
> Broken for me too since the latest AirTouch update. Along with the integration being broken, I also noticed the AirTouch 5 Android phone APP also has issues with fetching data and requires a close and re-open of the app on the console. This only temporarily fixed the phone app as shortly afterwards, the error appeared again. ![SmartSelect_20240627_063328_AirTouch 5](https://private-user-images.githubusercontent.com/42607382/343323731-2d343a1c-1e9f-4e68-aa0d-3421ab46fe66.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0NTU5NzMsIm5iZiI6MTcxOTQ1NTY3MywicGF0aCI6Ii80MjYwNzM4Mi8zNDMzMjM3MzEtMmQzNDNhMWMtMWU5Zi00ZTY4LWFhMGQtMzQyMWFiNDZmZTY2LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI3VDAyMzQzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0ZGM5ZTcxNWUyMWJjMWExY2NkY2JkNWJhMGFkZDE0NWRiOWNiNzM4ZDUxMDJlMTViNTQ2ZmQ4ODg4Mjc5MWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0gl82ltIOu6--O6Fk3mDZlHCl8Q0dl_sONtsqtFtAZI)

FYI - After some more testing, I've learnt the android phone app and LCD console stop working (the touchscreen is unresponsive on the LCD Console and the android APP shows the error demonstrated in the screenshot above) due to the HA Component still actively trying to initialise (although failing)... It seems that when the HA Integration attempts to initialise it causes the system to freeze requiring a reset of the LCD Console. Since disabling the HA Integration, the LCD Console and any remote APPs are working correctly.

**Comment by JohnMcIl** [comment]
I can confirm that this is NOT the case for me.

The latest iOS app still cannot connect regardless of the state of the HA integration.

AirTouch Australia confirms the fault is as a result of their software update and is affecting their mobile app.

John Mac

**Comment by JohnMcIl** [comment]
The HA integration is breaking the other comms for AirTouch as @pppaulie correctly identified.

**Comment by TheNoctambulist** [comment]
Thanks for the reports everyone.
Polyaire have pulled the update, but I've released v0.7.2 which should address the compatibility issues with the protocol and be more lenient for future updates. 
I'll look into whether there is anything I can do in the integration to avoid locking up the AirTouch console if any connection errors occur in the future.

Let me know if you still see any errors in the latest release.

**Comment by pppaulie** [comment]
Thank you for your prompt action resolving the issue 👏👏👏
Very much appreciated 👍
